plugins/woocommerce/includes/admin/class-wc-admin-api-keys.php

```diff
@@ -43,7 +43,7 @@ public function allow_save_settings( $allow ) {
 	 * @return bool
 	 */
 	private function is_api_keys_settings_page() {
-		return isset( $_GET['page'], $_GET['tab'], $_GET['section'] ) && 'wc-settings' === $_GET['page'] && 'advanced' === $_GET['tab'] && 'keys' === $_GET['section']; // WPCS: input var okay, CSRF ok.
+		return is_wc_admin_settings_page() && isset( $_GET['tab'], $_GET['section'] ) && 'advanced' === $_GET['tab'] && 'keys' === $_GET['section']; // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Note: same as original, but adds `is_admin()` check, which looks safe based on the usage context.